src/css/popup-new.css

```diff
@@ -42,6 +49,9 @@
   /* Bottom corners only */
   border-radius: 0 0 var(--borderRadiusSm) var(--borderRadiusSm);
   background-color: var(--colorWhite);
+  position: sticky;
+  top: 0;
+  z-index: 20;
```

Comment:
Ideally z-indexes are accompanied by a comment explaining what they're covering, so future contributors know whether they can be removed and/or what z-indexes to use for other elements.

Reply:
Welp, good news. I went and tried it without a Z-index and it worked without anything. 💣 Revised in 4021b7f61fa14cabcfea18a7169a3c7ffcc1b4ab